tests/ts_simple_recompile_ddl.erl

```diff
@@ -0,0 +1,87 @@
+%% -------------------------------------------------------------------
+%%
+%% Copyright (c) 2015 Basho Technologies, Inc.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
+
+-module(ts_simple_recompile_ddl).
+
+-behavior(riak_test).
+
+-include_lib("eunit/include/eunit.hrl").
+
+-export([confirm/0]).
+-define(TABLE, ?MODULE).
+-define(KV_MODULE, riak_kv_compile_tab).
+
+confirm() ->
+    {Cluster, _Conn} = ts_util:cluster_and_connect(single),
+    Node = hd(Cluster),
+    lists:foreach(
+        fun(Table) ->
+            DDL = create_table_sql(Table),
+            ts_util:create_and_activate_bucket_type(Cluster, DDL, Table)
```

Comment:
Great question. Perhaps because it wasn't there originally when the testing infrastructure was built?  It would be an easy change, but let's do that in a separate PR.